### DISCUSSION TITLE
Requesting clarification on 'read performance benchmark' instructions

### DISCUSSION THREAD

**Discussion by geetasg** [etcd-io/etcd#15322] [Q&A]
The benchmark results listed here - https://etcd.io/docs/v3.4/op-guide/performance/ - have following 

benchmark --endpoints=${HOST_1},${HOST_2},${HOST_3} --conns=100 --clients=1000 \
    range YOUR_KEY --consistency=l --total=100000

Here what is the size of YOUR_KEY? Is this referring to the keys added by write performance steps above (size 256)?

**Comment by SimFG** [comment]
1. The `YOUR_KEY` is random, which is generated by write performance steps above. 
2. Yes. The size of the key is 8byte and the size of value is 256byte.
![image](https://user-images.githubusercontent.com/21985684/219314115-d5cc81b8-a7c8-477d-842c-f3293faad171.png)
